xmtp_mls/src/groups/mod.rs

```diff
@@ -440,10 +440,12 @@ impl MlsGroup {
         let message_id = self.prepare_message(message, &conn);
 
         // Skipping a full sync here and instead just firing and forgetting
-        if let Err(err) = self.publish_intents(conn, client).await {
+        if let Err(err) = self.publish_intents(conn.clone(), client).await {
```

Comment:
Why the new clone here? I don't see anything changed about publish_intents

Reply:
`publish_intents()` consumes the connection, and we are using it again below in `sync_until_all_intents_resolved()`. This seems to be a common pattern, see `maybe_update_installations()` above, where the conn is cloned

Reply:
Ahh, I see it now